### DISCUSSION TITLE
Can you skip videos on export?

### DISCUSSION THREAD

**Discussion by odedia** [RhetTbull/osxphotos#1535] [General]
I tried to find the parameter in https://rhettbull.github.io/osxphotos/cli.html but didn't find such `--skip` option.

**Comment by RhetTbull** [comment]
`--only-photos`

**Reply by RhetTbull** [reply]
This will still export Live Photo video components I think. Use `--skip-live` to skip the videos.

**Reply by odedia** [reply]
Ahh… was searching for a “skip” parameter 😬. Thanks!

**Reply by RhetTbull** [reply]
Yeah, naming things is hard! It might be worth adding `--skip` options for these.
